client/app/projects/projects.html

```diff
@@ -5,7 +5,7 @@
     <div id="header">
     <h1> {{ past ? "Past" : "Current" }} RCOS Projects</h1>
 
-    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#addProject">
+    <button type="button" class="btn btn-primary" data-toggle="modal" data-target="#addProject" ng-show = "userOnProject">
```

Comment:
Can you remove the spaces between ng-show, =, and the string? HTML generally keeps things attached with no spaces between assignment of variables.